app/scripts/controllers/mobileClients.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+/**
+ * @ngdoc function
+ * @name openshiftConsole.controller:MobileClientsController
+ * @description
+ * Controller of the openshiftConsole
+ */
+angular.module('openshiftConsole')
+  .controller('MobileClientsController',
+    function ($scope,
+              $filter,
+              $routeParams,
+              Constants,
+              DataService,
+              ProjectsService) {
+
+      $scope.projectName = $routeParams.project;
+      $scope.name = $routeParams.deployment;
```

Comment:
Are all these variables used?

Reply:
projectName is being used, I ll remove name var